collections/concepts/what-are-tokens.md

```diff
@@ -1,32 +1,78 @@
+nav_order: 2 
+has_children: true 
+has_toc: false 
 image:
     path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
-    width: 1200
+    width: 1200 
     height: 630
 ---
 
-# What are tokens?
+# What are Tokens?
 
-Tokens are the core of the Basis Theory platform, built to enable companies to remove the need to store sensitive data while granting the flexibility they need to grow and operate their businesses. Our tokens enable you to either own the encryption keys yourself or pass raw data to our platform, and we will handle keeping it safe for you.
+Tokens are the core of the Basis Theory platform, built to enable companies to remove the need to store sensitive data
+while granting the flexibility they need to grow and operate their businesses. Our tokens enable you to either own the
+encryption keys yourself or pass raw data to our platform, and we will handle keeping it safe for you.
 
 ## How is the data stored?
 
 We enable our customers to send us data in two ways, unencrypted and pre-encrypted.
 
 ### You encrypt the data
 
-With privacy in mind, we've built our platform to enable you to encrypt all of your sensitive data yourself using one of our SDKs, Proxy, or your own internal encryption services. This allows your business to quickly and easily store your encrypted data in a separate air-gapped system, greatly reducing the liability of your encryption keys and data living in the same systems.
+With privacy in mind, we've built our platform to enable you to encrypt all of your sensitive data yourself using one of
+our SDKs, Proxy, or your own internal encryption services. This allows your business to quickly and easily store your
+encrypted data in a separate air-gapped system, greatly reducing the liability of your encryption keys and data living
+in the same systems.
 
 ### Basis Theory encrypts the data
 
-When you tokenize unencrypted data with Basis Theory, we will carefully encrypt the data with industry standard encryption algorithms. We ensure the data has been encrypted with a one-time use encryption key, which is then encrypted again and stored within our platform. This foundational encryption ensures your data is uniquely encrypted each time a new token is created. We will never mix our customers' encryption keys. Your keys are only used for your data, period.
+When you tokenize unencrypted data with Basis Theory, we will carefully encrypt the data with industry standard
+encryption algorithms. We ensure the data has been encrypted with a one-time use encryption key, which is then encrypted
+again and stored within our platform. This foundational encryption ensures your data is uniquely encrypted each time a
+new token is created. We will never mix our customers' encryption keys. Your keys are only used for your data, period.
+
+## What types of data can I tokenize?
+
+You can tokenize any type of data with Basis Theory - from one of our pre-defined data types to arbitrary unstructured

```

Comment:
> from one of our pre-defined data types

Are these "data types" or "token types"? If the latter, should we also link back to the Token Types documentation?